README.md

```diff
@@ -12,11 +12,10 @@ Yet another [HTTPie](https://httpie.io/) clone in Rust.
 brew install ht-rust
 ```
 
-### On Arch Linux via AUR
-[ht-bin AUR package](https://aur.archlinux.org/packages/ht-bin)
+### On Arch Linux via Pacman
 
 ```
-yay -S ht-bin
+pacman -S ht

```

Comment:
Can you add a comment mentioning the name of the binary that will be installed?

Reply:
Sure, see b73bd09f3094da6e9900fc822d4b50d387e93c53.

Reply:
103f88b